### PR TITLE
added the link for the documentation for glossary for the project

### PR DESCRIPTION
**Pull request by venegasgabby76**
Pull Request Template
Description
Updated the read me file to include a link to the proper glossary for the project

Commits Checklist
 commits have clear title names
 commit has descriptive message of what has changed

Loom Video:
https://www.loom.com/share/f6da1a2d7eaf46fa8283dc6ec5083988

Trello card:
https://trello.com/c/BF4zR3uW/500-add-glossary-information-to-readme-frontend

PRs needs to be reviewed by at least 2 team members
TPL is the 3rd reviewer
Pull Request Rubric

### CODE REVIEW

**Review by casey-dietz — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -252,6 +251,8 @@ Remember that this project is licensed under the MIT license, and by submitting
 
 ## Documentation
 
+- This is the link to the proper glossary documentation for the project (https://docs.google.com/document/d/1I2dtnjUtSmGxIfcuUWKrIRBhrCAFlmQ0RnJ3DZJIy4g/edit)
```
Comment (Aaron-TheCreator) [inline README.md:254]: This will clear up SO MUCH confusion surrounding terms! Thank you Gabby!

**Review by Aaron-TheCreator — APPROVED** [review]
Thanks for staying through that meeting to get this information and updating it for future devs 🙏🏾

**Comment by Aaron-TheCreator** [comment]
looks like package-lock.json has a conflict, remember to never push this file

**Review by ashtilawat — COMMENTED** [review]
Looks like the package.json was sent up with the README. I can't merge this because we need to maintain the same package throughout the project. I would go "git add filename" and not commit the package.json. You might have to make a new branch, do another git add and send up that branch.

**Review by ashtilawat — CHANGES_REQUESTED** [review]
Remove package-lock.json from commit.

**Pull request closed without merging**